ios/lottie_native.podspec

```diff
@@ -14,7 +14,7 @@ Pod::Spec.new do |s|
   s.source_files        = 'Classes/**/*'
 
   s.dependency 'Flutter'
-  s.dependency 'lottie-ios', '~> 4.4.3'
+  s.dependency 'lottie-ios', '4.5.1'
```

Comment:
Why is it necessary to require a fixed version? The removed `~>` constraint already allowed `4.5.1`